independent-projects/resteasy-reactive/common/processor/src/main/java/org/jboss/resteasy/reactive/common/processor/EndpointIndexer.java

```diff
@@ -1334,6 +1334,7 @@ && isParameterContainerType(paramType.asClassType())) {
                 if (field) {
                     return builder;
                 }
+                builder.setName(sourceName);
```

Comment:
@geoand I had to add this to know the parameter name.

Reply:
Makes sense